src/Codec/CBOR/Cuddle/CDDL/Resolve.hs

```diff
@@ -191,6 +191,18 @@ buildRefCTree rules = CTreeRoot $ fmap toCTreeRule rules
       toCTreeT0 t0
     toCTreeT2 (T2Tag (Just tag) t0) =
       It . CTree.Tag tag $ toCTreeT0 t0
+    toCTreeT2 (T2DataItem 7 (Just 20)) =
+      It $ CTree.Literal (VBool False)
+    toCTreeT2 (T2DataItem 7 (Just 21)) =
+      It $ CTree.Literal (VBool True)
+    toCTreeT2 (T2DataItem 7 (Just 25)) =
+      It $ CTree.Postlude PTHalf
+    toCTreeT2 (T2DataItem 7 (Just 26)) =
+      It $ CTree.Postlude PTFloat
+    toCTreeT2 (T2DataItem 7 (Just 27)) =
+      It $ CTree.Postlude PTDouble
+    toCTreeT2 (T2DataItem 7 (Just 23)) =
+      It $ CTree.Postlude PTUndefined
```

Comment:
Maybe nicer to extract this to its own function?

Reply:
Done